Assets/Scenes/Garden.unity

```diff
@@ -57021,6 +57005,7 @@ MonoBehaviour:
   _leanTowardsSurface: .5
   _glideHorizontalSpeed: 5
   _glideDescentRate: 1
+  _glideTurnRate: .5
 --- !u!114 &1215378093
 MonoBehaviour:
   m_ObjectHideFlags: 0
```

Comment:
They're automatic scene changes that are the result of changing what members the components have. It doesn't matter if they're squashed because the values are preserved in the prefabs.